layouts/_default/summary.html

```diff
@@ -0,0 +1,9 @@
+<span style="text-align: center;">
```

Comment:
wdyt about moving this to `partials` folder?

Reply:
not sure I get this - you want me to move the whole summary as a partial or just the span here?

Reply:
I meant moving the whole summary :)